src/Controllers/PoolProviderController.cs

```diff
@@ -371,26 +371,29 @@ public async Task<IActionResult> GetAgentRequestStatus([FromBody] AgentRequestSt
                 case "passed":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Work is in progress or complete. (Helix work item in job {correlationId} for agent {workItemId} was picked up by machine {workItemDetails.MachineName} and is {workItemDetails.State})"
+                        statusMessage = $"Work is in progress or complete. (Helix work item in job {correlationId}, work item {workItemId} queued in " +
+                                        $"{agentRequestStatusItem.agentData.queueId} was picked up by machine {workItemDetails.MachineName} and is {workItemDetails.State})"
                     });
                 case "unscheduled":
                     return Json(new AgentStatusItem()
                     {
-                        statusMessage = $"Work has been submitted to the Helix API, but has not started yet (Helix work item in job {correlationId} for agent {workItemId} is currently in state '{workItemDetails.State}')"
+                        statusMessage = $"Work has been submitted to the Helix API ({agentRequestStatusItem.agentData.queueId}), but has not started yet " +
+                                        $"(Helix work item in job {correlationId}, work item {workItemId} is currently {workItemDetails.State}, all machines seem to be busy)"
```

Comment:
Suggestion: "all machines seem to be busy"  -->  "machines are either busy or provisionining"

Reply:
@MattGal FYI you can use the suggestion button and suggest the change directly:
![image](https://user-images.githubusercontent.com/7013027/92720883-8c574f00-f365-11ea-9e42-1a019cc99e76.png)

Like this:
```suggestion
                                        $"(Helix work item in job {correlationId}, work item {workItemId} is currently {workItemDetails.State}, machines are either busy or provisionining)"
```